src/Page/NotAlone.elm

```diff
@@ -29,6 +32,79 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ div [] [ text "Not Alone page" ]
+        [ header []
+            [ h1 [ css [ pageHeading ] ] [ text "[cCc] You Are Not Alone" ]
+            ]
+        , div [ css [ grid ] ]
+            [ card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "22"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "33"
+            , card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "62"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "33"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "55"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "44"
+            ]
+        , div [] [ text "Not Alone page" ]
         , a [ href "definition" ] [ text "Find out more about Economic Abuse" ]
         ]
+
+
+grid : Style
+grid =
+    batch
+        [ displayFlex
+        , flexWrap wrap
+        , justifyContent spaceBetween
+        ]
+
+
+card : String -> String -> String -> Html msg
+card quote name age =
```

Comment:
Will this be the whole card? Or `cardFront`?

Reply:
Whole card for now :)

Reply:
There will be 2 more quotes but we can abstract once they start taking Keys.